### ISSUE TITLE
add get energy script

### ISSUE THREAD

**Issue by felipeZ** [MD-Studio/cerise-mdstudio-das5#8]
Once the [gromit script](https://github.com/MD-Studio/cerise-mdstudio-das5/blob/develop/mdstudio/gromit/gromit_mpi.sh) finishes we want to collect the energies using the [getEnergies script](https://github.com/MD-Studio/MDStudio/tree/prototype/components/lie_md/lie_md/scripts). It would  be great if you wan include the `getEnergies` script into the specialization.

**Comment by LourensVeen** [comment]
Hold on, shouldn't this be a separate step?

**Comment by felipeZ** [comment]
Yes, I agree But I don't know how to add it. Could you please give me a hand?

**Comment by felipeZ** [comment]
I will add another CWL step
